meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/AnalysisMojo.java

```diff
@@ -179,4 +189,4 @@ private static File[] collectInputFiles(FuzzFramework framework) throws IOExcept
         files.sort(Comparator.comparingLong(File::lastModified));
         return files.toArray(new File[0]);
     }
-}
\ No newline at end of file
+}
```

Comment:
vim? :)

Reply:
Yes... I'll see how I could remove the newline.